packages/gatsby-cli/package.json

```diff
@@ -22,11 +22,14 @@
     "fs-extra": "^4.0.1",
     "gatsby-telemetry": "^1.0.4",
     "hosted-git-info": "^2.6.0",
+    "ink": "^2.0.5",
+    "ink-spinner": "^3.0.0",
     "lodash": "^4.17.10",
     "meant": "^1.0.1",
     "node-fetch": "2.3.0",
     "opentracing": "^0.14.3",
     "pretty-error": "^2.1.1",
+    "react": "^16.8.4",
```

Comment:
react is a dependency of cli because ink needs it. Because cli can be installed globally a Peer dependency won't work and we won't use this version in the browser or anything so the user still needs to install it's own version on their site.

Reply:
React — boldly going where no JS framework has ever gone before 👩‍🚀